pkg/skaffold/config/options.go

```diff
@@ -33,14 +33,26 @@ type WaitForDeletions struct {
 // SkaffoldOptions are options that are set by command line arguments not included
 // in the config file itself
 type SkaffoldOptions struct {
-	ConfigurationFile     string
-	ConfigurationFilter   []string
-	HydratedManifests     []string
-	GlobalConfig          string
-	EventLogFile          string
-	RenderOutput          string
-	User                  string
+	ConfigurationFile string
+	Command           string
+	GlobalConfig      string
+	EventLogFile      string
+	RenderOutput      string
+	User              string
+	CustomTag         string
+	Namespace         string
+	CacheFile         string
+	Trigger           string
+	KubeContext       string
+	KubeConfig        string
+	DigestSource      string
+	// TODO(https://github.com/GoogleContainerTools/skaffold/issues/3668):
+	// remove minikubeProfile from here and instead detect it by matching the
+	// kubecontext API Server to minikube profiles
+	MinikubeProfile       string
+	RepoCacheDir          string
 	Apply                 bool
+	Debug                 bool
```

Comment:
So we had this before and removed it to instead use the RunMode.  But there is a difference between a mode and and command, so I'm alright with keeping this.

This name is hard to distinguish from `-v debug` or some internal debugging flag though.  WDYT of `ContainerDebugging`?

Reply:
yeah I thought about using the RunMode but it didn't seem the same. I'm fine with changing the name